skribenten-web/frontend/src/routeTree.gen.ts

```diff
@@ -13,11 +13,11 @@
 import { Route as rootRoute } from './routes/__root'
 import { Route as IndexImport } from './routes/index'
 import { Route as SaksnummerIndexImport } from './routes/saksnummer.index'
-import { Route as SaksnummerSaksIdImport } from './routes/saksnummer_/$saksId/route'
-import { Route as SaksnummerSaksIdBrevvelgerImport } from './routes/saksnummer_/$saksId/brevvelger/route'
+import { Route as SaksnummerSaksIdRouteImport } from './routes/saksnummer_/$saksId/route'
```

Comment:
hmm, må finne ut hvorfor tanstack hele tiden liker å gjøre endringer i denne filen for meg...